source/_docs/https.md

```diff
@@ -104,6 +106,8 @@ Refer to [Cloudflare Domain Configuration](/docs/cloudflare/).
 
 
 ## Known Issues
+### HTTPS doesn't provision with incorrect AAAA configurations
+Pantheon cannot not begin provisioning HTTPS if the Site Dashboard detects incorrect values set on AAAA records. Once you update the records using the recommended values, HTTPS will start to provision automatically. The values for AAAA records look similar, but they are distinct.
```

Comment:
"Once you update the AAAA records to the recommended values, contact Pantheon support to trigger HTTPS provisioning."